### ISSUE TITLE
Syntax Error: def get_access_token(self) -> str:

### ISSUE THREAD

**Issue by patriksharma** [argrento/huami-token#26]
I have cloned the repo and installed python on my MAC OS version 10.15.5

When I try to run the command it keep showing me this error message in terminal 

```
MacBook-Pro:huami-token patrik$ python huami_token.py --method xiaomi --bt_keys
  File "huami_token.py", line 46
    def get_access_token(self) -> str:
                               ^
SyntaxError: invalid syntax
```

**Comment by Skiti** [comment]
Did you install and use Python3?
I have Python 3.8.6 and run it with the command "python3 huami_token.py --method xiaomi --bt_keys"

**Comment by argrento** [comment]
Yes. `->` is a python3 feature.

**Comment by patriksharma** [comment]
Okay So I was on python 2.7.16 earlier. I decided to update it to latest version. 
then I used this command to update python

`brew install python3 && cp /usr/local/bin/python3 /usr/local/bin/python`
` brew upgrade python@3.9`

I have tried with both commands and both are showing different error. see attached screenshot from Terminal window

![Screenshot 2021-02-19 at 8 50 21 AM](https://user-images.githubusercontent.com/383524/108453117-cf600b80-728f-11eb-9660-16a9c1601c13.png)
